src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$card-width: 200px;
+$card-height: 300px;
+$header-height: 60px;
+$card-margin-bottom: 48px;
+$card-margin-right: 48px;
```

Comment:
There was no need to create a variables for those values. They were used only once.